pyproject.toml

```diff
@@ -72,8 +72,8 @@ dependencies = [
     "timm>=0.4.12",
     # torch 1.13+ required by torchvision
     "torch>=1.13",
-    # torchmetrics 0.10+ required for binary/multiclass/multilabel classification metrics
-    "torchmetrics>=0.10",
+    # torchmetrics 1.2.0+ required for MetricCollection
```

Comment:
MetricCollection exists in older releases too, this isn't the reason why 1.2.0+ is required. Also, I usually chop off the .0 at the end

Reply:
I could keep iterating on older versions until the test errors appear, then ref those/dig into the reason. 1.2 was released a year ago, I think it is a reasonable expectation that codebases will be updated since then? https://pypi.org/project/torchmetrics/#history

Reply:
I would still like to know why 1.2 is required. I would start by searching for the error message you were seeing on GitHub.